src/main/java/net/sf/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -36,6 +36,28 @@ public String getKey() {
 
     @Override
     public String format(String value) {
+        // Fixes https://github.com/JabRef/jabref/issues/1181
+        // Handle case names in order lastname, firstname and separated by ","
+        // E.g., Ali Babar, M., Dingsøyr, T., Lago, P., van der Vliet, H.
+        String[] valueParts = value.split(",");
+        if ((valueParts.length % 2) == 0) {
+            // We hit the described special case
+            StringBuilder sb = new StringBuilder();
+            for (int i = 0; i < valueParts.length; i++) {
+                sb.append(valueParts[i]);
+                if ((i % 2) == 0) {
+                    // Hit separation between last name and firstname --> comma has to be kept
+                    sb.append(',');
+                } else {
+                    // Hit separation between full names (e.g., Ali Babar, M. and Dingsøyr, T.) --> semicolon has to be used
+                    // Will be treated correctly by AuthorList.parse(value);
+                    sb.append(';');
+                }
+            }
+            value = sb.toString();
+        } else {
+            // No special treatment needed, proceed normally
```

Comment:
This makes no sense to me. Then you would have to comment each if without an else branch?!